js_modules/dagster-ui/packages/ui-core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -224,6 +224,10 @@ const LaunchAssetChoosePartitionsDialogBody = ({
     (!launchWithRangesAsTags && keysFiltered.length !== 1);
 
   const backfillPolicyVaries = assets.some((a) => a.backfillPolicy !== assets[0]?.backfillPolicy);
+  const isUnsupportedBackfillPolicyContext =
+    target.type === 'job' &&
+    !isHiddenAssetGroupJob(target.jobName) &&
+    assets.some((a) => a.backfillPolicy !== null); // at least one defined backfillPolicy
```

Comment:
This looks good to me, thanks for using a nice descriptive var name 😅 

When we launched the backfillPolicies feature I remember we were talking about removing this "null" case so that manually defining the default policy on your asset didn't cause different behavior than letting the default be inferred.  Seems like a good thing we never did that since the `no policy` case actually behaves differently in the context of asset jobs!